cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsResourceConnector.java

```diff
@@ -422,7 +422,7 @@ private void resumeAutoScaling(AmazonAutoScalingRetryClient amazonASClient, Coll
                 new Parameter().withParameterKey("CBUserData").withParameterValue(stack.getImage().getUserDataByType(InstanceGroupType.CORE)),
                 new Parameter().withParameterKey("CBGateWayUserData").withParameterValue(stack.getImage().getUserDataByType(InstanceGroupType.GATEWAY)),
                 new Parameter().withParameterKey("StackName").withParameterValue(stackName),
-                new Parameter().withParameterKey("StackOwner").withParameterValue(ac.getCloudContext().getOwner()),
+                new Parameter().withParameterKey("StackOwner").withParameterValue(ac.getCloudContext().getUserId()),
```

Comment:
I thought we need the user name here. Or is the user id the email address here?

Reply:
Yes, userid is the user's e-mail address. The owner was the uuid.

Reply:
cool, thank you